conf/defaults.ini

```diff
@@ -985,6 +985,13 @@ enabled = true
 # Enable the Profile section
 enabled = true
 
+#################################### News #############################
+[news]
+# Enable the news feed section
+news_feed_enabled = false
+# Feed used in the news drawer. Default to https://grafana.com/blog/news.xml
+news_feed_url = 
+
```

Comment:
Hold off on merging with these settings, they could be classified as part of custom branding (enterprise setting). Need to confirm this.